tests/test_autoscaler.py

```diff
@@ -408,3 +409,14 @@ def test_chunks():
     assert list(chunks([1], 1)) == [[1]]
     assert list(chunks([1, 2], 1)) == [[1], [2]]
     assert list(chunks([1, 2, 3], 2)) == [[1, 2], [3]]
+
+
+class FlaskTest(unittest.TestCase):
```

Comment:
you don't need `unittest.TestCase`, just use the usual py.test style as all other tests do (simple functions)